### PR TITLE
Fix: Error Sidebar PR failed test

### PR DESCRIPTION
**Pull request by Dedelweiss**
The purpose of this Pull request is to correct my PR https://github.com/lampepfl/dotty/pull/17229
Sorry for this failure.
I have tested the following command locally and my tests pass.
`sbt ";dist/pack ;scala3-bootstrapped/compile ;scala3-bootstrapped/test"`

[test_windows_full]

Fixes: #17963

### CODE REVIEW

**Comment by Dedelweiss** [comment]
> Thanks for the pr @Dedelweiss! So just to clarify the issue was with the `println`? Or what was actually causing the failure? Also I see you removed `-title`, is that related or a separate issue?

Hello Chris, thank you for your feedback. The main problem is that I didn't delete the title of the example with an error (NoTitle), so my assert was wrong for loadSidebarNoTitleError(). So I think that if there's a problem, it would probably come from there. The println is just an Int I noticed.

**Comment by Dedelweiss** [comment]
I also added the [test_windows_full] in the body of my commit so I can test the full test run here.

File: scaladoc/test/dotty/tools/scaladoc/site/SidebarParserTest.scala [hunk 1]
```diff
@@ -38,10 +38,9 @@ class SidebarParserTest:
       |          - page: my-page6/my-page6/my-page6.md
       """.stripMargin
 
-  private val sidebarNoTitle =
+  private val sidebarErrorNoTitle =
     """index: index.md
       |subsection:
-      |  - title: My title
       |    page: my-page1.md
```
Comment (julienrf) [inline scaladoc/test/dotty/tools/scaladoc/site/SidebarParserTest.scala:44]: This change makes the YAML invalid, I think there should be a `-` before `page`.

Also, I am surprised that the regression occurred only on Windows. Do you know why?

**Review by ckipp01 — APPROVED** [review]
Thanks for fixing this @Dedelweiss. Since we're now removing the actual check for the page (which you added in your other PR), if you do fix this and still want to add it in a way that won't error, please do!

**Comment by Dedelweiss** [comment]
> Thanks for fixing this @Dedelweiss. Since we're now removing the actual check for the page (which you added in your other PR), if you do fix this and still want to add it in a way that won't error, please do!

Sure, thank you

**Pull request merged**